### PR TITLE
Disable jemalloc on OpenBSD and MSVC where it is not working

### PR DESCRIPTION
**Pull request by nazar-pc**
Fix for issue introduced by https://github.com/paritytech/parity-common/pull/629 on MSVC target: https://github.com/subspace/subspace/runs/5612067150?check_suite_focus=true

### CODE REVIEW

**Review by ordian — APPROVED** [review]
Sorry about breaking your CI and thanks for the fix!

It wasn't caught because we're not testing kvdb-rocksdb on windows: https://github.com/paritytech/parity-common/blob/70a745a701f1917dd9c54a009c46d00f6fcb1dee/.github/workflows/ci.yml#L146

FYI: the plan is to deprecate rocksdb backend in favor of https://github.com/paritytech/parity-db.

**Comment by nazar-pc** [comment]
> FYI: the long-term plan is to deprecate rocksdb backend in favor of https://github.com/paritytech/parity-db.

I know and can't wait for it :slightly_smiling_face:

File: kvdb-rocksdb/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+- Disable `jemalloc` feature for `rocksdb` where it is not working. [#633](https://github.com/paritytech/parity-common/pull/633)
```
Comment (ordian) [inline kvdb-rocksdb/CHANGELOG.md:8]:
```suggestion

## [0.15.2] - 2022-03-20
- Disable `jemalloc` feature for `rocksdb` where it is not working. [#633](https://github.com/paritytech/parity-common/pull/633)
```

**Comment by ordian** [comment]
Published to crates.io

**Pull request merged**